ReactiveTask/Task.swift

```diff
@@ -43,6 +43,14 @@ public struct TaskDescription {
 		self.environment = environment
 		self.standardInput = standardInput
 	}
+	
+	/// A GCD group which to wait completion
+	private static var group = dispatch_group_create()
+	
+	/// wait for all task termination
+	public static func waitForAllTaskTermination() {
```

Comment:
Can we do this automatically before notifying `launchTask`'s observer of success/failure? In other words, we could automatically wait for termination before sending completed/error.